Source/Orts.Simulation/Simulation/Physics/Train.cs

```diff
@@ -11611,6 +11609,23 @@ public void TemporarilyRemoveFromTrack()
             activeActions.Clear();
         }
 
+        //================================================================================================//
+        //
+        // Checks if it has to go to next active subpath
+        //
+        public void TryIncrementSubpath()
+        {
+            // active subpath must be incremented in parallel in incorporated train if present; not just after incorporation
+            if (IncorporatedTrainNo >= 0)
+            {
+                var incorporatedTrain = Simulator.TrainDictionary[IncorporatedTrainNo];
+                if (incorporatedTrain.PresentPosition[0].TCSectionIndex != PresentPosition[1].TCSectionIndex && incorporatedTrain.PresentPosition[1].TCSectionIndex != PresentPosition[1].TCSectionIndex)
+                    IncrementSubpath(incorporatedTrain);
+                incorporatedTrain.PresentPosition[0].TCSectionIndex = -1;
+                incorporatedTrain.PresentPosition[0].TCSectionIndex = -1;
```

Comment:
```suggestion
                incorporatedTrain.PresentPosition[1].TCSectionIndex = -1;
```
Did you mean to duplicate the assignment like this? Perhaps you meant to make the second index 1, or is this just a duplicate line?